stdlib/bigdecimal/0/big_decimal.rbs

```diff
@@ -885,3 +885,47 @@ BigDecimal::SIGN_POSITIVE_ZERO: Integer
 # The version of bigdecimal library
 #
 BigDecimal::VERSION: String
+
+module Kernel
+  private
+
+  # Create a new BigDecimal object.
+  #
+  # initial
+  # :   The initial value, as an Integer, a Float, a Rational, a BigDecimal, or a
+  #     String.
+  #
+  #     If it is a String, spaces are ignored and unrecognized characters
+  #     terminate the value.
+  #
+  # digits
+  # :   The number of significant digits, as an Integer. If omitted or 0, the
+  #     number of significant digits is determined from the initial value.
+  #
+  #     The actual number of significant digits used in computation is usually
+  #     larger than the specified number.
+  #
+  # exception
+  # :   Whether an exception should be raised on invalid arguments. `true` by
+  #     default, if passed `false`, just returns `nil` for invalid.
+  #
+  #
+  # #### Exceptions
+  #
+  # TypeError
+  # :   If the `initial` type is neither Integer, Float, Rational, nor BigDecimal,
+  #     this exception is raised.
+  #
+  # TypeError
+  # :   If the `digits` is not an Integer, this exception is raised.
+  #
+  # ArgumentError
+  # :   If `initial` is a Float, and the `digits` is larger than Float::DIG + 1,
+  #     this exception is raised.
+  #
+  # ArgumentError
+  # :   If the `initial` is a Float or Rational, and the `digits` value is
+  #     omitted, this exception is raised.
+  #
+  def self?.BigDecimal: ((real | string | BigDecimal) initial, ?int digits, ?exception: bool) -> BigDecimal
```

Comment:
> __[Goodcheck]__ Prefer `boolish` over `bool` for method arguments and block return values ([view](https://sider.review/gh/repos/174796313/pulls/817#issue-6217627))
>
> <details>
>
> See the doc below:
> https://github.com/ruby/rbs/blob/78d04a2db0f1c4925d2b13c2939868edf9465d6c/docs/syntax.md#bool-or-boolish
>
> | Rule |
> | ---- |
> | `rbs.prefer_boolish` |
>
> You can [close this issue](https://sider.review/gh/repos/174796313/pulls/817#issue-6217627) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).
>
> </details>

I closed this issue. Reason:
- When you strictly want `true | false`. Use `bool` in this case.